src/test/java/training/AppTest.java

```diff
@@ -1,14 +1,77 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
-package training;
-
-import org.junit.Test;
-import static org.junit.Assert.*;
-
-public class AppTest {
-    @Test public void testAppHasAGreeting() {
-        App classUnderTest = new App();
-        assertNotNull("app should have a greeting", classUnderTest.getGreeting());
+package Training;
+
+import java.util.Scanner;
+
+abstract class AppTest {
+  abstract void findTriangle(double b, double h);
+  abstract void findRectangle(double l, double b);
+  abstract void findSquare(double s);
+  abstract void findCircle(double r);
+}
+
```

Comment:
findArea cannot be a class. Classes should be noun and should not be verbs. 

You need to use inheritance properly

Reply:
ok i will rewrite and update